sys/include/crypto/3des.h

```diff
@@ -85,8 +85,8 @@ static const uint32_t bigbyte[24] = {
  * @param   keySize     the size of the key
 
  *
- * @return  Whether initialization was successful. The command may be
- *          unsuccessful if the key size is not valid.
+ * @return  CIPHER_INIT_SUCCESS if the initialization was successful.
+ *          The command may be unsuccessful if the key size is not valid.
```

Comment:
> If the key size is not valid the *_init functions return CIPHER_ERR_INVALID_KEY_SIZE. The BAD_CONTEXT SIZE is returned if the alogorithm's define is missing in the Makefile (or the include)

true, 
still I would vote to have the `CIPHER_ERR_BAD_CONTEXT_SIZE` return value commented.
I propose then something like:
``` C
@return CIPHER_ERR_BAD_CONTEXT_SIZE if CIPHER_MAX_CONTEXT_SIZE has not been defined
```